examples/example.py

```diff
@@ -0,0 +1,14 @@
+"""Example client for PyTado"""
+
+from PyTado.interface.interface import Tado
+
+
+def main() -> None:
+    """Retrieve all zones, once successfully logged in"""
+    tado = Tado(username="mail@email.com", password="password")  # nosec
+    zones = tado.get_zones()
+    print(zones)
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
even if this change seems to not be related to the issue of the PR, would you mind adding a small information to the README.md to what this folder is used for? I personally would be interested, why you ignored the example_dev.py?

Reply:
Sure, I can update the readme.
I ignored the `example_dev.py` which is my local used function to test functionality against my personal account. If I need to recreate this and add my credentials every time the container is build, I find that annoying - basically it allows me to develop faster and not work around keeping my development setup functional.